src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/movement/fly/modes/specific/FlyClip.kt

```diff
@@ -88,6 +87,12 @@ object FlyNcpClip : Choice("NcpClip") {
             waitUntil { collidesVertical() }
 
             if (clipping != 0f) {
+                network.sendPacket(
+                    PlayerMoveC2SPacket.PositionAndOnGround(
+                        player.x, player.y, player.z,
+                        false
+                    )
+                )
```

Comment:
Why this?

Reply:
sending an additional packet can help with movement, because of vanilla setbacks due to moving too fast without sending an additional packet.